packages/perseus/src/types.ts

```diff
@@ -589,6 +595,12 @@ export type WidgetExports<
      */
     scorer?: WidgetScorerFunction;
 
+    /**
+     * A function that provides a public version of the widget options that can
+     * be shared with the client.
+     */
+    splitWidgetOptions?: SplitWidgetOptionsFunctions;
```

Comment:
I think the doc string here reinforces my point above about needing to rename this concept/field. What do you think of this suggested naming?

```suggestion
    /**
     * A function that provides a public version of the widget options that can
     * be shared with the client.
     */
    removePublicWidgetOptions?: RemovePublicWidgetOptionsFunction;
```

Reply:
Hmm, I think you mean `removePrivateWidgetOptions`, right? It started out as `getPublicWidgetOptions` before it was changed to "split", which is why I use that phrasing in the docstring.

Reply:
🤦‍♂️ Yes! "private" not "public"! :)